main.py

```diff
@@ -1 +1,25 @@
 #Mad Libs Generator Project
+#Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
This is causing your code to loop through itself 8 times. This is unnecessary since each variable is going to be prompting the user to input their word!

Reply:
this is a comment about a comment